src/lib.rs

```diff
@@ -387,29 +393,21 @@ mod tests {
 
     #[test]
     fn anonymous_bytes_cipher() {
-        let data = generate_random_string(50);
+        let data = generate_random_bytes(50);
         let sk = SecretId::new();
         let sk2 = SecretId::new();
         let pk = sk.public_id();
 
         let ciphertext = pk.encrypt_anonymous_bytes(&data);
         assert_ne!(&ciphertext, &data);
 
-        let error_res: Result<_, _> = sk.decrypt_anonymous_bytes(&data);
-        match error_res {
-            Err(_e) => (),
-            Ok(_) => {
-                panic!("Unexpected result: we're using wrong data, it should have returned error")
-            }
-        }
+        let _ = sk
+            .decrypt_anonymous_bytes(&data)
+            .expect_err("Unexpected result: we're using wrong data, it should have returned error");
```

Comment:
@nbaksalyar Sorry man, i guess these came from review that @Fraser999 added and i overlooked this (incorrect) review point of his. `expect_err()` belongs to `unwrap()/expect()` etc family of functions which we deliberately avoid because it does not give us the line numbers etc and no backtrace in release. Hence the usage of macros like `unwrap!()/panic!()` - so what you did earlier was correct and i spoke to fraser and he agrees - also in line with our style guide. I would let this pass for this iteration but there's another review i added so makes sense to just address this anyway.

Reply:
Sorry - my bad! :(